lib/g_searcher_web/controllers/api/search_result_controller.ex

```diff
@@ -3,8 +3,9 @@ defmodule GSearcherWeb.API.SearchResultController do
 
   alias GSearcher.{SearchResults, SearchResultURLs}
   alias GSearcherWeb.ErrorHandler
+  alias GSearcherWeb.Validators.{ParamsValidator, SearchResultParams}
 
-  def index(conn, _params) do
+  def index(conn, params) when params == %{} do
```

Comment:
How about?
```suggestion
  def index(conn, %{}) do
```

Reply:
Putting a blank map there is actually a valid pattern-match even if there are params 😄 
![Screen Shot 2021-07-06 at 10 58 05 AM](https://user-images.githubusercontent.com/34730459/124540232-2afaf200-de49-11eb-9c61-fa0b1988a56c.png)

We're kind of forced to do it this way, or maybe something like `map_size(params) == 0` 🤔